parser-typechecker/src/Unison/TermPrinter.hs

```diff
@@ -368,7 +368,7 @@ pretty0
            -> ([Pretty SyntaxText] -> Pretty SyntaxText)
            -> Pretty SyntaxText
   printLet elideUnit sc bs e im uses =
-    paren ((sc /= Block) && p >= 12)
+    paren ((sc /= Block) && p >= 3)
```

Comment:
This will cause parens to print around a `let` block if it appears as an argument to a function.